tests/Unit/Evolution/Systems/GeneralizedHarmonic/Test_UpwindFlux.cpp

```diff
@@ -0,0 +1,547 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+#include "tests/Unit/TestingFramework.hpp"
+
+#include <array>
+#include <cstddef>
+#include <limits>
+#include <random>
+#include <utility>
+
+#include "ApparentHorizons/Strahlkorper.hpp"
+#include "ApparentHorizons/StrahlkorperGr.hpp"
+#include "ApparentHorizons/Tags.hpp"  // IWYU pragma: keep
```

Comment:
Why is the numerical flux test using code from the horizon finder?

Reply:
Because I'm testing the flux on a Strahlkorper, and I need to get the Strahlkorper's normal

Reply:
I don't see anything about a `Strahlkorper` so I am still confused by why you need things from ApparentHorizons